src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1914,7 +1914,17 @@ private AssemblySymbol GetForwardedToAssembly(string fullName, int arity, out bo
 
         internal static void CheckFeatureAvailability(Location location, MessageID feature, DiagnosticBag diagnostics)
         {
-            LanguageVersion availableVersion = ((CSharpParseOptions)location.SourceTree.Options).LanguageVersion;
+            var options = (CSharpParseOptions)location.SourceTree.Options;
+            string featureFlag = feature.RequiredFeature();
+            if (featureFlag != null)
```

Comment:
Is this the only usage of `featureFlag`? If so, could you just inline the expression?